charts/pega/README.md

```diff
@@ -782,7 +782,8 @@ Use the `installer` section  of the values file with the appropriate parameters
 Parameter   | Description   | Default value
 ---         | ---           | ---
 `image`   | Reference the `platform/installer` Docker image that you downloaded and pushed to your Docker registry that your deployment can access.  | `YOUR_INSTALLER_IMAGE:TAG`
-`adminPassword` | Specify a temporary, initial password to log into teh Pega application. This will need to be changed at first login. The adminPassword value cannot start with "@". | `"ADMIN_PASSWORD"`
+`imagePullPolicy` | Specify when to pull an image. | `IfNotPresent`
```

Comment:
The intro paragraph to these parameters doesn't mention this and I don't believe clients have enough context to which "when" refers. Can you explain this context in the paragraph leading up to this table? you might have to also provide an example so it's clear what this new "image pull" policy is all about.

Reply:
Or we just don't document this feature :)